mne/viz/utils.py

```diff
@@ -1685,10 +1677,17 @@ def _compute_scalings(scalings, inst):
         data = inst._data.swapaxes(0, 1).reshape([len(inst.ch_names), -1])
     # Iterate through ch types and update scaling if ' auto'
     for key, value in scalings.items():
-        if value != 'auto':
-            continue
         if key not in ch_types.keys():
-            raise ValueError("Sensor {} doesn't exist in data".format(key))
+            continue
+        if not (isinstance(value, str) and value == 'auto'):
+            try:
+                scalings[key] = float(value)
```

Comment:
Is there a use case where scaling values aren't given as floats? Just curious...

Reply:
It should always either be float like or 'auto'

Reply:
(by float like I mean they could give an int, or some numpy float, and it's nice to know that whatever they give we'll have a standard float down the line)